app/services/courses/content_status_service.rb

```diff
@@ -0,0 +1,23 @@
+module Courses
+  class ContentStatusService
+    def execute(enrichment, next_cycle_boolean)
```

Comment:
```suggestion
    def execute(enrichment, next_recruitment_cycle?)
```
Any reason you didn't go with this?

Reply:
After speaking to Dan he thinks it's better to pass in the cycle to show the dependancy